jest.config.js

```diff
@@ -14,5 +14,21 @@ module.exports = {
     '/tests/e2e/',
     '/build/',
   ],
-  reporters: ['default', 'jest-junit'],
+  coverage: true,
```

Comment:
I think this should be `collectCoverage`

Reply:
oh yes you're right!! I changed it now!